projects/sft-protocol/index.js

```diff
@@ -40,7 +43,9 @@ module.exports = {
     filecoin: {
         tvl: async (_, _1, _2, { api }) => {
 
-            let balances = {};
+            const lendingPoolTvl = await api.call({ target: LENDING_POOL_ADDRESS, abi: TOTAL_SUPPLY_ABI, params: [] });
+            api.add(ADDRESSES.null, lendingPoolTvl);
```

Comment:
why is totalSupply added to tvl and also token balance in lending pool? shouldnt we include only the token balance in lending pool? isnt the borrowed amount already counted under active miners?

Reply:
> why is totalSupply added to tvl and also token balance in lending pool? shouldnt we include only the token balance in lending pool? isnt the borrowed amount already counted under active miners?

hey guy, please check again and merge it if no problem.